<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>QG_PolylineOptions</class>
  <widget class="QWidget" name="QG_PolylineOptions">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>650</width>
        <height>22</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy>
        <hsizetype>4</hsizetype>
        <vsizetype>5</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="minimumSize">
      <size>
        <width>650</width>
        <height>22</height>
      </size>
    </property>
    <property name="maximumSize">
      <size>
        <width>650</width>
        <height>22</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Polyline Options</string>
    </property>
    <layout class="QHBoxLayout">
      <property name="margin">
        <number>1</number>
      </property>
      <item>
        <widget class="QToolButton" name="bClose">
          <property name="text">
            <string>Close</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QToolButton" name="bUndo">
          <property name="text">
            <string>Undo</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QComboBox" name="cbMode">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <item>
            <property name="text">
              <string>Line</string>
            </property>
          </item>
          <item>
            <property name="text">
              <string>Tangential</string>
            </property>
          </item>
          <item>
            <property name="text">
              <string>Tan Radius</string>
            </property>
          </item>
          <item>
            <property name="text">
              <string>Angle</string>
            </property>
          </item>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="lRadius">
          <property name="text">
            <string>Radius:</string>
          </property>
          <property name="alignment">
            <set>Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="leRadius"/>
      </item>
      <item>
        <widget class="QLabel" name="lAngle">
          <property name="text">
            <string>Angle:</string>
          </property>
          <property name="alignment">
            <set>Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="leAngle"/>
      </item>
      <item>
        <widget class="Q3ButtonGroup" name="buttonGroup1">
          <property name="lineWidth">
            <number>0</number>
          </property>
          <property name="title">
            <string/>
          </property>
          <property name="flat">
            <bool>true</bool>
          </property>
          <widget class="QRadioButton" name="rbNeg">
            <property name="geometry">
              <rect>
                <x>38</x>
                <y>2</y>
                <width>36</width>
                <height>19</height>
              </rect>
            </property>
            <property name="text">
              <string/>
            </property>
            <property name="icon">
              <pixmap>image0</pixmap>
            </property>
            <property name="toolTip" stdset="0">
              <string>Clockwise</string>
            </property>
          </widget>
          <widget class="QRadioButton" name="rbPos">
            <property name="geometry">
              <rect>
                <x>2</x>
                <y>2</y>
                <width>36</width>
                <height>19</height>
              </rect>
            </property>
            <property name="text">
              <string/>
            </property>
            <property name="icon">
              <pixmap>image1</pixmap>
            </property>
            <property name="checked">
              <bool>true</bool>
            </property>
            <property name="toolTip" stdset="0">
              <string>Counter Clockwise</string>
            </property>
          </widget>
        </widget>
      </item>
      <item>
        <widget class="Line" name="sep1">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="frameShape">
            <enum>QFrame::VLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <images>
    <image name="image0">
      <data format="PNG" length="172">89504e470d0a1a0a0000000d494844520000000e0000000f0806000000d414fe7400000073494441542891bd93510ac0200c4353f152e6fe64c7721fc321a562ddc7f259fad216a349c2a91ad9ab2f92ec51b3241b1000540f8d86c870762c190800fc1a2503b5607d03700c01eec648d7c2b444c58cfe0549f6ef1325d92a2dab6992ac004f9c32f0fcdeefaa3bd887c4fcefd8857ce80672d24274c76563db0000000049454e44ae426082</data>
    </image>
    <image name="image1">
      <data format="PNG" length="164">89504e470d0a1a0a0000000d494844520000000e0000000f0806000000d414fe740000006b494441542891bd924b0e80200c44a78d97a2f7cf782c5c61c8584275e1db11fad2af91c44c44742490b4f97da8a401ab3faf4823e35c8d57a44c36002549cbf67d58ce2791a4fd9b11903d66b4c541b8ee4739171377e0b9dc8a7cf7b893d5b4b7473e7a7e88155a44bf0011fe441424295e9a0000000049454e44ae426082</data>
    </image>
  </images>
  <includes>
    <include location="local">rs_line.h</include>
    <include location="local">rs_settings.h</include>
    <include location="local">rs_actiondrawpolyline.h</include>
  </includes>
  <connections>
    <connection>
      <sender>bClose</sender>
      <signal>clicked()</signal>
      <receiver>QG_PolylineOptions</receiver>
      <slot>close()</slot>
    </connection>
    <connection>
      <sender>bUndo</sender>
      <signal>clicked()</signal>
      <receiver>QG_PolylineOptions</receiver>
      <slot>undo()</slot>
    </connection>
    <connection>
      <sender>leRadius</sender>
      <signal>textChanged(QString)</signal>
      <receiver>QG_PolylineOptions</receiver>
      <slot>updateRadius(QString)</slot>
    </connection>
    <connection>
      <sender>leAngle</sender>
      <signal>textChanged(QString)</signal>
      <receiver>QG_PolylineOptions</receiver>
      <slot>updateAngle(QString)</slot>
    </connection>
    <connection>
      <sender>cbMode</sender>
      <signal>activated(int)</signal>
      <receiver>QG_PolylineOptions</receiver>
      <slot>updateMode(int)</slot>
    </connection>
    <connection>
      <sender>rbNeg</sender>
      <signal>toggled(bool)</signal>
      <receiver>QG_PolylineOptions</receiver>
      <slot>updateDirection(bool)</slot>
    </connection>
    <connection>
      <sender>rbPos</sender>
      <signal>toggled(bool)</signal>
      <receiver>QG_PolylineOptions</receiver>
      <slot>updateDirection(bool)</slot>
    </connection>
  </connections>
</ui>
